### PR TITLE
Fix tests post cruft update and fix RtD build

### PR DESCRIPTION
**Pull request by huard**
## Overview

This PR fixes Travis tests failures and RtD build failure.

Changes:

* Fix ERA5 data fetch. 
* Fix pep8

## Related Issue / Discussion
#271 

## Additional Information

Links to other issues or sources.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -252,8 +252,7 @@ pep8:
 docs:
 	@echo "Generating docs with Sphinx ..."
 	@-bash -c '$(MAKE) -C $@ clean html'
-	@echo "Opening browser to: file:/$(APP_ROOT)/docs/build/html/index.html"
-	@-bash -c 'xdg-open $(APP_ROOT)/docs/build/html/index.html'
+	@echo "Open browser to: file:/$(APP_ROOT)/docs/build/html/index.html"
```
Comment (tlvu) [inline Makefile:255]: This was coming from the cookiecutter, I'll backport.

**Comment by tlvu** [comment]
Confirmed RtD build fixed https://readthedocs.org/projects/pavics-raven/builds/11096212/

File: docs/source/conf.py [hunk 2]
```diff
@@ -53,6 +53,9 @@
 autoapi_file_pattern = '*.py'
 autoapi_options = ['members', 'undoc-members', 'private-members']
 
+# We are using mock imports in readthedocs, so probably safer to not run the notebooks
+nbsphinx_execute = 'never'
```
Comment (tlvu) [inline docs/source/conf.py:57]: Will backport this one to cookiecutter too as it fixes RtD build.

**Comment by tlvu** [comment]
@huard TravisCI build status update to Github seems to be bugging today (the status is stuck in "in progress") but it is completed and does pass.  You can merge this PR so RtD build is fixed.

**Comment by tlvu** [comment]
@huard You squash merge this PR, it that accidental or intentional?  I feel like there was nothing that needed to be squashed in this PR.

**Comment by huard** [comment]
I did not really think about it, and that was the default setting... Should the default be to not squash ?

**Comment by tlvu** [comment]
> Should the default be to not squash ?

I think yes.  There are very few cases that we actually need squashing.

Squashing gives a more linear git history but we lose a huge amount of useful details.

It's akin to squashing all the bugs fixes and new features between the releases.  How are we going to know which changes is for which bug fix or new feature !

There are different levels of "aggregations" to handle the different amount of details.  We do not need to squash to manage ourselves the level of details.  The PR "merge commit" aggregates all the little individual commits, the release note aggregates all the different bug fixes and/or new features (PR) in between the releases.

**Comment by Zeitsperre** [comment]
I would build this discussion to https://github.com/Ouranosinc/xclim/issues/373

I think it boils down to a case-by-case basis. For PRs that are solely bug fixes and have a ton of small commits *cough*tomakeTravisCIwork*cough*, I'm a fan of squashing to prevent them from polluting the history. I do see the value in keeping commits separated though. Would it make sense to not squash by default but keep that option available?

**Comment by tlvu** [comment]
Ha !  I was actually looking for that Ouranosinc/xclim#373 issue to link here.

Yes I did not meant to never squash, I simply meant squashing should not be the default.

**Pull request merged**